manim/mobject/mobject.py

```diff
@@ -1140,6 +1427,34 @@ def put_start_and_end_on(self, start, end):
 
     # Background rectangle
     def add_background_rectangle(self, color=BLACK, opacity=0.75, **kwargs):
+        """Add a BackgroundRectangle as submobject.
+
+        The BackgroundRectangle is added behind other submobjects.
+
+        This can be used to increase the mobjects visibility in front of a noisy background.
+
+        Parameters
+        ----------
+        color : str
```

Comment:
this should point to https://github.com/ManimCommunity/manim/blob/master/manim/utils/color.py#L103

Reply:
But wouldn't that remove the possibility to pass a custom color like func("#123456") ?